docs/monoDrive_home/Simulator-Commands.md

```diff
@@ -0,0 +1,365 @@
+# Simulator API Commands
+
+## Get Simulator Version
+Retrieves the current version of the simulator and of the simulator API with the commands documented on this page.
+
+**command ID**:  "GetVersion"
+
+**command data**:
+```json
+{	
+}
+```
+**response**: (string)
```

Comment:
I think these responses are actually going to be JSON objects that come back. Not sure if this is accurate.

Reply:
yes they will be, but they are not in 1.11 for most commands